### PR TITLE
Patrick/add router

### PR DESCRIPTION
**Pull request by patrick-fs**
- Adding a router for product, cart, checkout, and thank you pages
- adding S3 configuration for static sites

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -1,8 +1,28 @@
 import React from 'react';
+import Products from './Products';
```
Comment (van-fs) [inline src/components/App.tsx]: Something I tend to do is create [barrel files](https://hackernoon.com/react-project-architecture-using-barrels-d086146eb0f6) for organization - typically for components, services, directives, etc.  It reduces the number of import statements.

File: src/index.tsx [hunk 2]
```diff
@@ -2,6 +2,12 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
 import App from './components/App';
+import { BrowserRouter as Router } from 'react-router-dom';
```
Comment (van-fs) [inline src/index.tsx:3]: I also tend to group import statements - like all external dependencies first then internal files.  (Technically I also alphabetize em in their groups but maybe that's overkill ;) ).  Food for thought, no need to change code.

**Review by van-fs — APPROVED** [review]
LGTM, added some stylistic comments just for discussion purposes.

File: src/components/App.tsx (hunk 1 above)
Comment (patrick-fs) [reply src/components/App.tsx]: I like that a lot!  Let me build out those page-level components a little more. I can see them being refactored into a "page barrel"

`<page.Cart />`
`<page.ThankYou />`

etc.

File: src/index.tsx (hunk 2 above)
Comment (patrick-fs) [reply src/index.tsx:3]: Good idea. Done!

**Pull request merged**